src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -135,9 +135,24 @@ pub enum CsrConnector {
     },
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum ReaderSchemaSelectionStrategy {
+    Latest,
+    Inline(String),
+    ById(i32),
+}
+
+impl Default for ReaderSchemaSelectionStrategy {
+    fn default() -> Self {
+        Self::Latest
+    }
+}
+
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct CsrConnectorAvro<T: AstInfo> {
     pub connector: CsrConnector,
+    pub key_strategy: Option<ReaderSchemaSelectionStrategy>,
+    pub value_strategy: Option<ReaderSchemaSelectionStrategy>,
```

Comment:
I think it makes sense to allow both, since this will be allowed in a bare `FORMAT`, which is translated into both a key and value Avro encoding, as discussed here: https://materializeinc.slack.com/archives/C01CFKM1QRF/p1656951710669979

Once we get rid of all this weirdness in key/value format/encoding (see same discussion), we should re-evaluate and do whatever makes sense.